tests/server/routes/cart-routes-test.js

```diff
@@ -199,10 +200,9 @@ describe('Cart routes', function(){
     });
 
     it('should increment a quanitity when POST route called a product that already exists in the cart', function (done) {
-      _loggedInAgent.put('/api/cart/')
+      _loggedInAgent.put('/api/cart/' + _cartItem._id )
```

Comment:
@karn09 is it more RESTful to have the cartitem id passed in rather than putting it on the url?